tests/integration_tests/test_applications.py

```diff
@@ -368,6 +368,15 @@
             "corsika",
         ],
     ],
+    "print_array_elements::print_compact_cors_telescopeheights": [
```

Comment:
Is "cors" here short for corsika? I suggest to change to corsika in that case.

Reply:
Fixed.